apps/src/weblab2/CDOIDE/cdo-ide-context/cdo-ide-context-config-reducer.ts

```diff
@@ -0,0 +1,7 @@
+import {ConfigType, ReducerAction} from '@cdoide/types';
```

Comment:
Can you give all non-css files camel cased names (react components should stay pascal cased)?

Reply:
done.